### PR TITLE
Add Rules Page

### PR DESCRIPTION
**Pull request by varun13169**
image

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
- Code changes looks good
- UI looks clean
- Liked your approach on theme toggle with classname

Good to Merge

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                <div className="dui-nav-sch-act__search dui-inp-txt dui-inp-txt--search">
+                    <i className="dui-inp-txt__icon dui-inp-txt__icon--search">
+                        <svg xmlns="http://www.w3.org/2000/svg"
+                            className="dui-inp-txt__icon-content dui-inp-txt__icon-content--search icon icon-tabler icon-tabler-search"
+                            width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#059669" fill="none"
+                            stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="10" cy="10" r="7" />
+                            <line x1="21" y1="21" x2="15" y2="15" />
+                        </svg>
```
Comment (zoef-yt) [inline src/components/navbar/Navbar.jsx:22]: You can use [icones](https://icones.netlify.app/) to make lines of code less and increase reusability.

File: src/page/quizPage/QuizPage.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    <ul class="dui-ul quiz-card__options">
+                        <li class="quiz-card__option dui-util-bdr-radi-m dui-util-spc-pad-s">
+                            <p><span class="dui-util-fw-bld">1) </span>Option 1</p>
+                        </li>
+                        <li class="quiz-card__option dui-util-bdr-radi-m dui-util-spc-pad-s">
+                            <p><span class="dui-util-fw-bld">2) </span>Option 1</p>
+                        </li>
+                        <li class="quiz-card__option dui-util-bdr-radi-m dui-util-spc-pad-s">
+                            <p><span class="dui-util-fw-bld">3) </span>Option 1</p>
+                        </li>
```
Comment (zoef-yt) [inline src/page/quizPage/QuizPage.jsx:38]: Can be converted into react component as very little things are changing other things are repeating.

**Review by zoef-yt — APPROVED** [review]
Ui looks great. The code is good. Good work Varun good to merge.

**Pull request merged**